src/Earth4All/Earth4All.jl

```diff
@@ -898,6 +888,17 @@ inits[:Owner_consumption_G_dollar_per_y] = inits[:Permanent_owner_cash_inflow_G_
 inits[:Life_of_extra_CO2_in_atm_y] = p[:Life_of_extra_CO2_in_atm_in_1980_y] * inits[:OWeoLoCO2]
 inits[:Old_growth_removal_rate_1_per_y] = p[:OGRR_in_1980_1_per_y] * inits[:FFLReoOGRR]
 inits[:Life_expectancy_y] = ( ( p[:LEmax] - (p[:LEmax] - p[:LE_in_1980]) * exp(- p[:LEgamma] * (inits[:Effective_GDP_per_person_k_dollar_per_p_per_y] - p[:GDP_per_person_in_1980_k_dollar_per_p_per_y])) ) * ( 1 + p[:LEalfa] * ( inits[:Effective_GDP_per_person_k_dollar_per_p_per_y] - p[:GDP_per_person_in_1980_k_dollar_per_p_per_y] )) ) * inits[:Warming_effect_on_life_expectancy_1] * inits[:Life_expectancy_multipler_1]
+inits[:LE_at_60_y] =  inits[:Life_expectancy_y] - 60
+inits[:Deaths_Mp_per_y1] = p[:Dying_in_1980_Mp_per_y] * inits[:LE_at_60_y] / 10
+inits[:Deaths_Mp_per_y2] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y3] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y4] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y5] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y6] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y7] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y8] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y9] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y10] = inits[:Deaths_Mp_per_y1]
```

Comment:
Why here only var1 is multiplied by (delay/order) while for passing_20/40/60 it is done for all var?

Reply:
It is actually done for all variables. Notice that variables var2, ..., var10 are receiving the same initialisation as var1.

Reply:
yes true, I didn't realize that.